packages/@aws-cdk/aws-servicecatalogappregistry/lib/aspects/stack-associator.ts

```diff
@@ -5,6 +5,16 @@ import { ApplicationAssociator } from '../application-associator';
 import { SharePermission } from '../common';
 import { isRegionUnresolved, isAccountUnresolved } from '../private/utils';
 
+export interface StackAssociatorBaseProps {
+  /**
+  * Indicates if the target Application should be shared with the cross-account stack owners and then
+  * associated with the cross-account stacks.
+  *
+  * @default - false
```

Comment:
```suggestion
  * @default - true
```
since this is what the user expressed, no? If they configured this construct to be cross-account then they probably wanted this to be shared cross account; it's a silent failure if we ignore that.

Reply:
We will be setting this to `false` to accommodate the fact that by default we should assume not all customers would've set up their accounts for cross-account sharing. To setup cross-account sharing of AppRegistry Applications, they need to be part of the same AWS Organization, and the organization needs to enable sharing.